azure/services/scalesets/scalesets.go

```diff
@@ -655,9 +658,12 @@ func base64EncodedHash(vmss compute.VirtualMachineScaleSet) (string, error) {
 	// Setting Admin Password is not supported but an initial password is required for Windows
 	// Don't include it in the hash since it is generated and won't be the same each the spec is created (#1182)
 	tmpPass := vmss.VirtualMachineProfile.OsProfile.AdminPassword
+	tmpCustomData := vmss.VirtualMachineProfile.OsProfile.CustomData
```

Comment:
let's update the comment above to explain why we also don't include custom data

Reply:
Fixed.